src/components/Books.js

```diff
@@ -0,0 +1,27 @@
+import Book from './Book';
+
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      author: 'John Doe',
+      title: 'Lorem Ipsum.',
+    },
+    {
+      id: 2,
+      author: 'Jane Doe',
+      title: 'Dolor Sit Amet',
+    },
+  ];
+  return (
+    <>
+      <div className="books">
+        {books.map((book) => (
+          <Book key={book.id} title={book.title} author={book.author} />
+        ))}
+      </div>
+    </>
+  );
+};
+
+export default Books;
```

Comment:
- [ ] You want to consider implementing a form for adding a book. ⚒️ 
- [ ] There is an error about @popper/core, which is a dependency that bootstrap needs. 🔴 

![image](https://user-images.githubusercontent.com/79876959/145914563-50a702e7-ae1d-4ee5-8934-aaf84fcf9288.png)